config/locales/en.yml

```diff
@@ -2361,21 +2361,21 @@ en:
         edit:
           body_html: You cannot use this tool this year because you %{ineligible_reason}. <strong>Is this a mistake?</strong> <a href=%{prev_link}>Go back to correct.</a>
           ineligible_reason:
-            eligibility_529_for_non_qual_expense: withdrew money from a 529 college savings account and used it for a non-qualifying expense in 2023
+            eligibility_529_for_non_qual_expense: withdrew money from a 529 college savings account and used it for a non-qualifying expense in 2024
```

Comment:
is there a reason some of these are hard-coded? at first i thought it was just NY stuff but this is AZ

Reply:
good catch! yes ill update